blobstorage/templates/_helpers.tpl

```diff
@@ -8,7 +8,7 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/instance: {{ .Release.Name -}}
 {{- end -}}
 
-{{- define "hmcts.releaseName" -}}
+{{- define "hmcts.blobstorage.releaseName" -}}
```

Comment:
why does clash matter? as long as it has the same behaviour I thought it would work?

Reply:
Because the java chart includes .chart.name in it and this doesn't . If we want it to be consistent, we can make it a breaking change ?

Reply:
And because i have spent some time trying to debug issue :D

Reply:
Shouldn’t they be consistent?

Reply:
I would do that in 1.0.0 if needed, this i think should be non breaking